flink-dist/src/main/resources/flink-conf.yaml

```diff
@@ -145,7 +145,7 @@ jobmanager.execution.failover-strategy: region
 
 # The address to which the REST client will connect to
 #
-#rest.address: 0.0.0.0
+rest.address: localhost
```

Comment:
Doesn't this now mean that the user has to change two config options if she wants to start the `JobManager` on a machine using the standalone setup?

Reply:
Assuming this is ok, do we have to adjust our documentation to point this out now?

Reply:
Yes, in some cases.
For a purely local setup, or a cluster setup where submissions are submitted internally (like, a master node running the JM + client), no change should be required.

It is only necessary if submissions are to be accepted from the outside. In that case I would think it is also good to require both to be explicitly configured, because only the REST API should be accessible from the outside.

Reply:
Which documentation parts do we have to update if we enable `rest.address` by default?

Reply:
After thinking about that and looking at the docs, I think we better not do this now. I'm not sure anymore whether this is as easy as I thought; essentially we have to go over all jobmanager.rpc.address references in the docs and determine whether this is necessary for the REST API, and that's not something I want to do right now.

Reply:
I'll open a follow-up ticket; this is still something we should look into.

Reply:
I thought we wouldn't do this change? Why is it still contained in the PR?